flintrock/ec2.py

```diff
@@ -864,11 +868,17 @@ def launch(
     else:
         user_data = ''
 
+    if spot_request_duration is None:
+        spot_request_valid_until = datetime.utcnow() + timedelta(days=7)
+    else:
+        spot_request_valid_until = datetime.utcnow() + duration_to_timedelta(spot_request_duration)
```

Comment:
Instead of `utcnow()`, the Python docs recommend using a timezone-aware method like `now()`.

For example:

```python
>>> from datetime import datetime, timezone
>>> datetime.now(tz=timezone.utc)
datetime.datetime(2020, 10, 18, 22, 38, 26, 703948, tzinfo=datetime.timezone.utc)
```